pkg/store/bucket_e2e_test.go

```diff
@@ -207,10 +210,37 @@ func prepareStoreWithTestBlocks(t testing.TB, dir string, bkt objstore.Bucket, m
 	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
 	defer cancel()
 
+	// Check if the blocks are being loaded.
+	start := time.Now()
+	time.Sleep(time.Second * 1)
```

Comment:
Do we need a test for this? Do we test other metrics somewhere? To me personally it just feels like we're introducing more complexity to the test while maybe not gaining that much value.

Reply:
Well, this is one way to test if the newly added metric `thanos_bucket_store_blocks_last_loaded` is working as expected.

Reply:
Yes I understand the intention. I'm just not sure whether this needs testing (hence my question if we're testing other metrics). I'll leave the decision to you if other reviewers do not share this feeling.